src/apps/concepts/components/AddConceptToDictionaryIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Fab,
+    Tooltip
+} from "@material-ui/core";
+import {APIConcept} from "../types";
+
```

Comment:
Could you clean-up the blank lines and the spacing / indentation a bit? (We tend to use 2 spaces to indent on this project)